ckanext/switzerland/tests/helpers/mock_ftp_storage_adapter.py

```diff
@@ -22,25 +28,46 @@ def cdremote(self, remotedir=None):
     def get_remote_filelist(self, folder=None):
         if folder is None:
             folder = self.cwd
-        return self.filesystem.listdir(folder, files_only=True)
+        return self.filesystem.listdir(folder)
 
     def get_remote_dirlist(self, folder=None):
         if folder is None:
             folder = self.cwd
-        return self.filesystem.listdir(folder, dirs_only=True)
+        return self.filesystem.listdir(folder)
```

Comment:
The arguments `files_only` and `dirs_only` no longer exist for the `listdir` method. This doesn't actually affect how we use the `MockFTPStorageAdapter` in the tests, so I have just removed the arguments.